clang-tools-extra/test/clang-doc/namespace.cpp

```diff
@@ -0,0 +1,340 @@
+// RUN: clang-doc --format=html --output=%t/docs --executor=standalone %s
+// RUN: clang-doc --format=md --output=%t/docs --executor=standalone %s
```

Comment:
Why no `mkdir`? I presume because clang-doc will make it, but the test shouldn't rely on that behavior. Plus we want to be sure each run is isolated, so starting w/ `rm -rf %t && mkdir %t` is a good practice if you need a directory. also, probably don't need `docs`...